### DISCUSSION TITLE
Queued is a misleading status for workflows with multiple jobs

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#126632] [Actions]
### Select Topic Area

Product Feedback

### Body

- This was originally reported as https://github.com/community/community/discussions/54295, but for https://github.com/community/community/discussions/70478 a distinction was made between Question and Product Feedback. It is now being refiled as Product Feedback.

If GitHub needs to refile it again, I'd appreciate it if a GitHub employee were to do this.

### Body

I have a job: https://github.com/jsoref/cspell-dicts/actions/runs/4866124681

In the actions list https://github.com/jsoref/cspell-dicts/actions it shows:
<img width="1311" alt="image" src="https://user-images.githubusercontent.com/2119212/235794910-7842c89f-1de7-4450-a038-3c8b7c3b1211.png">

But for most of its duration, 2-3 of its jobs were running (apparently it now has 80 jobs in the matrix). It's true that some jobs were Queued, but it's probably more important to me that some of its jobs were running than that some weren't.

**Comment by D4M13N-D3V** [comment]
I agree

**Comment by seyahdoo** [comment]
Any updates on this? Im currently battling this problem and its workarounds really doesnt look nice.

**Comment by jsoref** [comment]
@nebuk89
